src/components/PopulationResultsViewer.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { Table } from '@mantine/core';
+import {
+  PopulationReportViewerProps,
+  DetailedMeasureReport,
+  MeasureReportInfoArray,
+  PopulationResult,
+} from '../types/PopulationResultsViewerTypes';
+
+export default function PopulationReportViewer({
+  reports,
+}: PopulationReportViewerProps) {
+  return (
+    <Table>
+      <thead>
+        <tr>
+          <th>Patient</th>
+          <th>Initial Population</th>
+          <th>Numerator</th>
+          <th>Denominator</th>
+          <th>Denominator Exclusion</th>
```

Comment:
Technically, these populations should not be hardcoded. We should be pulling off the populations from the groups within the measure report. Certain measures use different populations than these 4.

We should assemble the `th`s by looping through the group and grabbing the code.coding.code. Maybe we should prefer code.coding.display if defined, and if not fall back to code.coding.code